### DISCUSSION TITLE
No code databases

### DISCUSSION THREAD

**Discussion by WilliamDiakite** [ouvroir/labouvroir#163] [General]
Des outils dans le style de Common pour se donner idées et conseiller les collègues. À suivre et évaluer.

- [Anytype](https://anytype.io/en#features), [docs](https://doc.anytype.io/d/fundamental-concepts)
- [Notion](https://notion.io)
- [Airtable](https://www.airtable.com/)

**Comment by lenaMK** [comment]
discussin à la réunion de coordo du 11 octobre 2022
- quels sont les exports possibles? éviter au maximum le problème d'être enfermé dans des formats comme filemaker. Export en csv requiert du traitement 
- quoi qu'il arrive, il y a toujours des étapes de nettoyage de données → csv et open refine
- coût de la documentation / bonnes pratiques
- si les gens manquent de temps, même s'ils le regrettent après ça va arrive
- revenir au bons vieux fichiers excels. Problèmes devalidation et d'emploi adéquat d'excel, problème des choix multiples (tags)
- ajouter à excel des outils qui vérifient / valident et éventuellement visualisent les données pour "voir" ce qu'il se passe
- zotero
- utiliser notion pour visualiser les donées
- dépend du cas d'utilisation? avoir des données vs rassembler et diffuser les informations pour la recherche

demander à Ersy si on pourrait "étudier" ses données Notion? (export d'une bd plus grande et complexe, gestion des images? )

**Comment by lenaMK** [comment]
guide de bonnes pratiques pour utiliser, nettoyer et visualiser excel pour la solution la plus simple et no code

**Comment by lenaMK** [comment]
plugin pour browser pour améliorer l'UX/UI? comment offrir une alternative potable à l'UX sexy de notion/airTable
"excel fait peur" comment démocratiser sa pratique?  bridgine the gap du fichier word aux données
avoir des exemples pour montrer que ça marche, être capable dee voir le potentiel dans "prendre le temps" d'apprendre à utiliser quelque chose

**Comment by lenaMK** [comment]
Détails à propos de Anytype:  tech stack
- desktop: electron + typescript
- mobile: swiftt, kotlin
- network and storage: IPFS, ThreadsDB, go-lang, Libp2p
- objects as [protobuf files](https://medium.com/nerd-for-tech/protobuf-what-why-fcb324a64564) on top of ipfs (attention: protobuf = Google...)

licence initiale prévue: MIT @emchateau :) 
mes notes complètes sont sur [dahnotes](https://github.com/ouvroir/dahnotes/blob/master/anytype.md)

**Comment by lenaMK** [comment]
Serait-ce le graal?
Open source no-code database and Airtable alternative  https://baserow.io/

**Comment by emchateau** [comment]
Malheureusement, je ne crois pas.
C’est un modèle freemium. 
C’est ouvert mais une partie du logiciel est sous licence.
Salutations,

Emmanuel
